AardvarkTests/ARKLogDistributorTests.m

```diff
@@ -246,7 +246,7 @@ - (void)test_distributeAllPendingLogsWithCompletionHandler_informsLogObserversOf
 
 #pragma mark - Performance Tests
 
-- (void)test_logDistribution_performance;
+- (void)disabled_test_logDistribution_performance;
```

Comment:
Can we leave a comment here explaining why this is disabled? Maybe include in the comment what the failure we're seeing is?

Also: is this failure only on Github Workflow? Or also on Travis?

Reply:
I haven't seen it on Travis in this PR, but I saw it in the Travis build on #82. I think it must be related to the Xcode version, although I haven't been able to repro locally, so there's likely other factors involved as well.

Reply:
Addressed in e085b05